tls/s2n_psk.h

```diff
@@ -48,6 +48,7 @@ struct s2n_psk {
 S2N_RESULT s2n_psk_init(struct s2n_psk *psk, s2n_psk_type type);
 S2N_CLEANUP_RESULT s2n_psk_wipe(struct s2n_psk *psk);
 S2N_RESULT s2n_psk_clone(struct s2n_psk *new_psk, struct s2n_psk *original_psk);
+S2N_RESULT s2n_resumption_psks_remove(struct s2n_connection *conn);
```

Comment:
To fit with naming, you might want to rename this to "s2n_psk_params_remove_resumption_psks" and make it accept the connection's psk_params instead of the connection itself.